app/views/jobseekers/job_applications/build/_caption.html.slim

```diff
@@ -0,0 +1 @@
+.govuk-caption-m = t("jobseekers.job_applications.current_step", current: JobApplication.completed_steps[step] + 1, total: JobApplication.completed_steps.values.last + 2)
```

Comment:
I don't agree that the current step is always the one after the last completed step.

1. Start new application
2. Click 'save as draft' on first step
3. Continue the application
4. You are taken to the review step and can leap into any step (all incomplete)